example/yarn.lock

```diff
@@ -8459,6 +8500,13 @@ node-dir@^0.1.17:
   dependencies:
     minimatch "^3.0.2"
 
+node-fetch@2.6.7:
```

Comment:
I was pretty excited about the graphql & httpClient re-using the same axios instance as their fetch impl.  Not necessarily in this PR (in fact I think this is good as is), but how would you feel about experimenting with the [custom fetch](https://www.npmjs.com/package/graphql-request#using-a-custom-fetch-method) option graphql-request offers?  The reason the re-use excited me was to have a single place to a) attach the authorization header and also (potentially) b) a single place to detect 401s.

Reply:
That seems like really solid reasoning to me. I'll see if I can sneak that in on another PR.